builtin/mainmenu/tab_settings.lua

```diff
@@ -203,10 +200,15 @@ local function formspec(tabview, name, tabdata)
 
 	if core.settings:get("touchscreen_threshold") ~= nil then
 		tab_string = tab_string ..
-			"label[4.3,4.2;" .. fgettext("Touchthreshold: (px)") .. "]" ..
-			"dropdown[4.25,4.65;3.5;dd_touchthreshold;0,10,20,30,40,50;" ..
+			"label[4.25,3.5;" .. fgettext("Touch threshold (px):") .. "]" ..
+			"dropdown[4.25,3.95;3.5;dd_touchthreshold;0,10,20,30,40,50;" ..
 			((tonumber(core.settings:get("touchscreen_threshold")) / 10) + 1) ..
-			"]box[4.0,4.5;3.75,1.0;#999999]"
+			"]"
+	else
```

Comment:
This condition above not correct. `touchscreen_threshold` could be specified by accident on desktops because it is exposed in the Settings menu.

What would you think of `PLATFORM == "Android"` as check on L204 ?

Reply:
Touch is supported on Linux currently (if compiled right), so a platform check would be less correct.

Reply:
I see. That's quire unfortunate.

Reply:
The current "correct" solution would be to add an API that returns true if `USE_TOUCHSCREENGUI` is defined. (checking `touchscreen_threshold` is close but can be accidentally specified in the config as you said)
Though in the future touch support shouldn't require toggling it during build-time and anything depending on that will have to be redesigned anyway.

Reply:
Out of scope for this PR though?

Reply:
Definitely.